bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -56,11 +58,11 @@
 
     // Note -- synchronized though I guess this should always be called from
     // gui thread...
-    public static synchronized String formatBrightness(Magnitude mag) {
+    private static synchronized String formatBrightness(Magnitude mag) {
```

Comment:
Not to my knowledge, but I didn't change this... I think Rob may have written it this way, so I didn't question it 😆 . I can look into it further.